### PR TITLE
pywhat 3.4.0

### PR DESCRIPTION
**Pull request by harens**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/pywhat.rb [hunk 1]
```diff
@@ -16,6 +16,7 @@ class Pywhat < Formula
   end
 
   depends_on "python@3.9"
+  depends_on "rust" # for orjson
```
Comment (carlocab) [inline Formula/pywhat.rb:19]: We might also want a `maturin` dependency to skip having to build it? Both probably build-only deps though.

Comment (branchvincent) [reply Formula/pywhat.rb]: ah right, fixed

**Comment by cho-m** [comment]
`maturin` formula dependency doesn't seem to work and pip still tries to build a copy of `maturin`. I think this is similar situation as `poetry` build system. From local install attempt, the logs show:
```
       Compiling maturin v0.11.3 (/private/tmp/pip-install-4_q_fhsm/maturin_dc2f759bff9b49d69bd9e898b30e936e)
        Finished release [optimized] target(s) in 2m 40s
    {'reason': 'compiler-artifact', 'package_id': 'maturin 0.11.3 (path+file:///private/tmp/pip-install-4_q_fhsm/maturin_dc2f759bff9b49d69bd9e898b30e936e)', 'manifest_path': '/private/tmp/pip-install-4_q_fhsm/maturin_dc2f759bff9b49d69bd9e898b30e936e/Cargo.toml', 'target': {'kind': ['bin'], 'crate_types': ['bin'], 'name': 'maturin', 'src_path': '/private/tmp/pip-install-4_q_fhsm/maturin_dc2f759bff9b49d69bd9e898b30e936e/src/main.rs', 'edition': '2018', 'doc': True, 'doctest': False, 'test': True}, 'profile': {'opt_level': '3', 'debuginfo': None, 'debug_assertions': False, 'overflow_checks': False, 'test': False}, 'features': ['configparser', 'default', 'dirs', 'human-panic', 'log', 'pretty_env_logger', 'reqwest', 'rpassword', 'rustls', 'upload'], 'filenames': ['/private/tmp/pip-install-4_q_fhsm/maturin_dc2f759bff9b49d69bd9e898b30e936e/target/release/maturin'], 'executable': '/private/tmp/pip-install-4_q_fhsm/maturin_dc2f759bff9b49d69bd9e898b30e936e/target/release/maturin', 'fresh': False}
```

I think the only way to avoid this is to directly call `maturin build ...` and manually install the `orjson` resource.

For simplicity, we can just let `pywhat` compile a copy of `maturin` for build phase (it shouldn't add it to final virtualenv).

**Comment by harens** [comment]
Thanks all for your help. I noticed that @branchvincent opened an issue about this https://github.com/bee-san/pyWhat/issues/142, and a new release has been published with a potential fix. I'll therefore close this PR in favour of that.

Thanks for your work maintaining this amazing project. It's greatly appreciated. 👍

**Comment by branchvincent** [comment]
@harens thank you for the kind words and for your contributions! 🍻

**Pull request closed without merging**